toolchain/check/testdata/function/builtin/method.carbon

```diff
@@ -0,0 +1,97 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// AUTOUPDATE
+
+interface I {
+  fn F[self: Self](other: Self) -> Self;
+}
+
+impl i32 as I {
+  fn F[self: i32](other: i32) -> i32 = "int.add";
+}
```

Comment:
Should there be a test of:

```
interface I {
  fn F[self: Self](other: Self) -> Self = "int.add";
}
```

Reply:
Hm, we don't have any implementation or testing of defined-but-not-defaulted associated entities in interfaces at all. Added some.